resources/automate.rb

```diff
@@ -0,0 +1,117 @@
+#
+# Cookbook Name:: chef_stack
+# Resource:: automate
+#
+# Copyright 2016 Chef Software Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource_name 'chef_automate'
+default_action :create
+
+property :name, String, name_property: true
+property :channel, Symbol, default: :stable
+property :version, [String, Symbol], default: :latest
+property :config, String, required: true
+property :accept_license, [TrueClass, FalseClass], default: false
+property :enterprise, [String, Array], default: 'chef'
+property :license, String
+property :chef_user, String, default: 'workflow'
+property :chef_user_pem, String, required: true
+property :validation_pem, String, required: true
+property :builder_pem, String, required: true
+property :platform, String
+property :platform_version, String
+
+load_current_value do
+  # node.run_state['chef-users'] ||= Mixlib::ShellOut.new('chef-server-ctl user-list').run_command.stdout
+  # node.run_state['chef-orgs'] ||= Mixlib::ShellOut.new('chef-server-ctl org-list').run_command.stdout
+  # current_value_does_not_exist! unless node.run_state['chef-orgs'].index(/^#{org}$/)
+end
+
+action :create do
+  # https://github.com/chef/delivery/issues/469
```

Comment:
👍 
At the time it explained why the following config option was being pushed.  I think we can remove the following config too, as it only affects 1 version of automate, and the likelyhood of customers installing that version is small.